pkgs/applications/misc/cura/default.nix

```diff
@@ -1,26 +1,27 @@
-{ mkDerivation, lib, fetchFromGitHub, cmake, python3, qtbase, qtquickcontrols2, qtgraphicaleffects, curaengine, plugins ? [] }:
+{ mkDerivation, lib, fetchFromGitHub, cmake, python3, qtbase,
+ qtquickcontrols2, qtgraphicaleffects, curaengine, plugins ? [] }:
 
 mkDerivation rec {
   pname = "cura";
-  version = "4.7.1";
+  version = "4.8.0";
 
   src = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "Cura";
-    rev = version;
-    sha256 = "19an168iad3cb5w8i71c0wbr79qnz5qnpxqx1j6dgh64qz6ffn2r";
+    rev = "4.8";
+    sha256 = "060fqzspipm93ks0inrj7yrj5wmvkdfv8xaxrv22590yb9f95s9m";
   };
 
   materials = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "fdm_materials";
-    rev = version;
```

Comment:
Why duplicate the version instead of keeping a variable for it?

Reply:
Because the tag for release 4.8.0 is `4.8`. When they go back to using the release version as the tag (if they ever do), then this can be changed back.

Reply:
Ah OK, then please add a comment in the file(s) that says that, so that people who change in the future get it and it doesn't cargo-cult.

Reply:
I'm fixing the tag for 4.8.0. It should above all be consistent. Sorry about this.

Reply:
Great, thanks @Ghostkeeper

Reply:
Revisions have been updated to use the version variable